frontend/src/component/insights/hooks/usePlaceholderData.ts

```diff
@@ -9,7 +10,7 @@ type PlaceholderDataOptions = {
 
 export const usePlaceholderData = (
     placeholderDataOptions?: PlaceholderDataOptions,
-) => {
+): ChartData<'line', number[]> => {
```

Comment:
Are you sure this type is correct? I see that this data is also piped into `CreationArchiveChart` eventually, which is bar chart, so it feels weird to type this underlying method to line chart.